UPGRADE-1.13.md

```diff
@@ -538,4 +686,36 @@ List of affected classes:
                     subject: 'Email address verification'
    ```
 
-1 Extracted the section responsible for the `ShopBundle` from `@SyliusCore/Email/accountVerification.html.twig` and relocated it to `@SyliusShop/Email/verification.html.twig`.
+### Miscellaneous
+
+1. Using Guzzle 6 has been deprecated in favor of Symfony HTTP Client. If you want to still use Guzzle 6 or Guzzle 7,
+   you need to install `composer require php-http/guzzle6-adapter` or `composer require php-http/guzzle7-adapter`
+   depending on your Guzzle version.
+   Subsequently, you need to register the adapter as a `Psr\Http\Client\ClientInterface` service as the following:
+    ```yaml
+        services:
+            Psr\Http\Client\ClientInterface:
+                class: Http\Adapter\Guzzle7\Client # for Guzzle 6 use Http\Adapter\Guzzle6\Client instead
+    ```
+
+1. The `payum/payum` package has been replaced by concrete packages like `payum/core`, `payum/offline`
+   or `payum/paypal-express-checkout-nvp`. If you need any other component so far provided by `payum/payum` package, you
+   need to install it explicitly.
```

Comment:
Having a `Dependencies` section could be nice, though I don't know if we had a lot of them